src/apps/gas-tank/components/History/GasTankHistoryCard.tsx

```
@@ -0,0 +1,252 @@
+      switch (sortConfig.key) {
+        case 'date':
+          aValue = a.timestamp;
+          bValue = b.timestamp;
+          break;
+        case 'type':
+          aValue = a.type;
+          bValue = b.type;
+          break;
+        case 'amount':
+          // Sort algebraically: Top-up is positive, Spend is negative
+          const aAmount = parseFloat(a.usdcAmount);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)